src/spyglass/utils/dj_mixin.py

```diff
@@ -246,16 +243,20 @@ def fetch_pynapple(self, *attrs, **kwargs):
             for file_name in nwb_files
         ]
 
-    # ------------------------ delete_downstream_merge ------------------------
+    # ------------------------ delete_downstream_parts ------------------------
 
-    def _import_merge_tables(self):
-        """Import all merge tables downstream of self."""
+    def _import_part_masters(self):
```

Comment:
Don't think this is getting called in `delete_downstream_parts()`

running:
```python
from spyglass.common import Session
from spyglass.position.v1 import TrodesPosV1
# from spyglass.decoding import PositionGroup

from spyglass.ripple.v1 import RippleTimesV1
from spyglass.linearization.v1 import LinearizedPositionV1
from spyglass.mua.v1 import MuaEventsV1

key = {"nwb_file_name":"minirec20230622_.nwb"}
(TrodesPosV1 & key).delete_downstream_parts(dry_run=True)
```

gives the value error for unimported `PositionGroup`.  Trying to debug didn't show the code coming inside this function

Reply:
apologies for the runaround on this - You're right, it wasn't loading the cache before loading the graph. I think I've fixed it now